test/functional/validator-options.spec.ts

```diff
@@ -8,23 +8,20 @@ describe('validator options', () => {
     class MyClass {
       @IsNotEmpty()
       title: string = '';
-      isActive: boolean;
     }
 
     const model = new MyClass();
     model.title = '';
-    return validator
-      .validate(model, { skipMissingProperties: true, validationError: { target: false } })
-      .then(errors => {
-        expect(errors.length).toEqual(1);
-        expect(errors[0].target).toBeUndefined();
-        expect(errors[0].property).toEqual('title');
-        expect(errors[0].constraints).toEqual({ isNotEmpty: 'title should not be empty' });
-        expect(errors[0].value).toEqual('');
-      });
+    return validator.validate(model, { validationError: { target: false } }).then(errors => {
+      expect(errors.length).toEqual(1);
+      expect(errors[0].target).toBeUndefined();
+      expect(errors[0].property).toEqual('title');
+      expect(errors[0].constraints).toEqual({ isNotEmpty: 'title should not be empty' });
+      expect(errors[0].value).toEqual('');
+    });
   });
```

Comment:
These seem like irrelevant changes?

Reply:
@braaar The `isActive` and `skipMissingProperties` parts are not relevant to that test so I was just trying to clean it up. I can revert that part if you like?

Reply:
I would submit that as a separate cleanup PR

Reply:
Okay @braaar I've reverted the changes not directly related to this PR.